### PR TITLE
Allows users to filter on dimension columns

### PR DESCRIPTION
**Pull request by helrond**

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -18,4 +18,43 @@ function displayData() {
   }
 }
```
Comment (EcZX) [inline js/app.js]: You saw nothing.

File: index.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
   <title>space barnacle</title>
   <!-- CDNs should be used for production deployment -->
   <!-- <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.15/css/dataTables.bootstrap.min.css"> -->
-  <!-- <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7-dist/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous"> -->
+  <!-- <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous"> -->
```
Comment (eswanson611) [inline index.html]: Looks great, but maybe we could add a shiny buckle to this bootstrap? Thoughts?

File: index.html [hunk 3]
```diff
@@ -72,28 +74,33 @@ <h2 class="modal-title">Help</h2>
...
-            </tr>
-          </thead>
-          <tbody>
-          </tbody>
-        </table>
+        <div class="col-12">
+          <table id="results" class="table" width="100%">
+            <thead class="thead-default">
+              <tr>
+                <th>Name</th>
```
Comment (jwelbs) [inline index.html]: Perhaps we should change this to Box/Container Profile?

**Comment by jwelbs** [comment]
I am excited to see this table in action!

**Pull request merged**